### PR TITLE
Improve syntax of information page

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | See below. Code changes are minimal, removing `card-text`, adding `mb-0` and adding one if condition and alert. The big changed blocks are because of indentation change. Didn't touch any other logic.
| Type?             | refacto
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Related PRs       | 
| How to test?      | Check that `Advanced Parameters > Information` and the top of `Advanced Parameters > Logs` pages look good.
| Possible impacts? |  None

### Changes
- Added a missing alert if no overrides were found.
- Removed extra spacing after alerts and paragraphs.
- Removed additional `card-text` divs, those have no effect, `card-body` makes the padding.

### Screenshot
information

### CODE REVIEW

**Review by nicosomb — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig
Comment (l-delin) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig]:
```suggestion
        <p class="mb-0">{{ 'You must provide this information when reporting an issue on GitHub or on the forum.'|trans }}</p>
```

**Review by matthieu-rolland — COMMENTED** [review]
LGTM :+1:   
I'll wait for wording feedback before sending the PR in QA

(since` you added a new wording in the advanced parameters page)

**Review by matthieu-rolland — DISMISSED** [review]

**Comment by l-delin** [comment]
The wording has been checked 🎁

**Comment by Hlavtox** [comment]
@nicosomb @matthieu-rolland Ready ✅

**Review by nicosomb — APPROVED** [review]

**Comment by kpodemski** [comment]
According to the [Backward compatibility promise](https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md), we're not allowed to change classes in a minor version, but on this page, the impact should be either little or none. The only thing I can think of is someone altering the style or content of the blocks targeting the class directly...

What do you think @NeOMakinG ?

**Comment by Hlavtox** [comment]
@kpodemski It should be clarified, how much you can change HTML markup? :D

Snímek obrazovky 2022-12-21 132247

It personally think that the risks are minimal, because if somebody targets `card-text` utility class, it's awful practice.

**Comment by kpodemski** [comment]
I agree and I'd like to approve it  😜 I'm leaving an approval but QA after @NeOMakinG confirmation.

**Review by kpodemski — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by hibatallahAouadni — COMMENTED** [review]
Hello @Hlavtox 

Thanks for your PR 🚀 
LGTM, except that there's no space between lines in the page Informations, see the attached screenshot below:

image

Ping @NeOMakinG @PrestaShop/ux-team wdyt about this new display? 

Thanks!

**Comment by Hlavtox** [comment]
@hibatallahAouadni This is clearly written in PR description - `Removed extra spacing after alerts and paragraphs.` and a screenshot is attached.

And Valentin approved this, so why ping him again?

**Comment by hibatallahAouadni** [comment]
Hello @Hlavtox 

I know but this PR description (this modification) is not approved neither by Product Team nor UX Team.
So, I can't approve without PM or UX approval.

Thanks for your understanding 🙏

**Comment by ElodieOS** [comment]
Hi @Hlavtox! 
Is it negative to have overrides and file changes? 
If it's negative, then the success banner is a very good idea.
If it's not particularly negative to have them, then I think a simple text will do. 
Either way, you are right, it is important to inform the user and not leave anything empty :)

**Comment by FabienPapet** [comment]
@ElodieOS Overrides can extend PrestaShop features and rewrite them if needed. Therefore, it is good to not have any, because you can easily upgrade your PrestaShop to newer versions. If you have some overrides, it is not bad, but this can block future updates, as code can change, etc.

**Comment by ElodieOS** [comment]
Hello @FabienPapet! 
Thank you for your precision. In this case, I validate @Hlavtox's design proposition 

Thank you for your work :)

**Review by hibatallahAouadni — APPROVED** [review]
Well then, it's QA ✅ as the UX team approved 🎉 
Thanks @Hlavtox for your hard working and happy new year everyone 🎉

**Comment by FabienPapet** [comment]
Thank you @Hlavtox

**Pull request merged**
